python/ray/_private/runtime_env/packaging.py

```diff
@@ -50,12 +50,13 @@ def __new__(cls, value, doc=None):
     HTTPS = "https", "Remote https path, assumes everything packed in one zip file."
     S3 = "s3", "Remote s3 path, assumes everything packed in one zip file."
     GS = "gs", "Remote google storage path, assumes everything packed in one zip file."
+    NFS = "file", "NFS storage path, assumes everything packed in one zip file."
```

Comment:
Prefer to just call it `file://` to match smart_open and because it can be used more generally. We can suggest in docs that people use this for NFS.